util/zlib/deflater.c

```diff
@@ -84,7 +84,8 @@ int main(int argc, char* argv[])
         }
 
         /* display summary */
-        printf("Compressed %s (%d bytes) to %s (%d bytes)\n",
-                argv[1], inlen, argv[2], outlen);
+        printf("Compressed %s (%lu bytes) to %s (%lu bytes)\n",
+                argv[1], (unsigned long) inlen,
+                argv[2], (unsigned long) outlen);
```

Comment:
cc65's `printf()` family of functions know the `h, j, t,` and `z` size modifiers.  Those functions don't know the `hh` modifier.

Reply:
Fixed in rebased commit.